gnucash/gnome/dialog-sx-editor.c

```diff
@@ -1207,10 +1209,12 @@ gnc_ui_scheduled_xaction_editor_dialog_create (GtkWindow *parent,
                             TRUE, TRUE, 0 );
     }
 
-    gnc_register_gui_component( DIALOG_SCHEDXACTION_EDITOR_CM_CLASS,
+    id = gnc_register_gui_component( DIALOG_SCHEDXACTION_EDITOR_CM_CLASS,
                                 NULL, /* no refresh handler */
                                 sxed_close_handler,
                                 sxed );
```

Comment:
reindent the parameters and since you touched it, fix the whitespace around the parens (before, not after for the open and none for the closing).